### PR TITLE
5317 Add two questions to pre-qualification page

### PR DESCRIPTION
**Pull request by alexwllms**
@benlovell @benbarnett

### CODE REVIEW

File: app/views/principals/pre_qualification_form.html.erb [hunk 1]
```diff
@@ -1,47 +1,111 @@
...
-        <label class="l-pre-qualification-questions__label">
-          <%= f.radio_button :question_3, 0, class: 'l-pre-qualification-questions__input' %>
-          <%= t('registration.answer_no') %>
-        </label>
-      </fieldset>
+    <%= heading_tag(t('registration.heading'), level: 1) %>
+    <p class="l-pre-qualification-questions__intro-text"><%= t('registration.introduction') %></p>
+
+    <%= form_for @prequalification, url: prequalify_principals_path, html: { class: 'form' } do |f| %>
+      <%= f.form_row :question_1 do %>
```
Comment (benbarnett) [inline app/views/principals/pre_qualification_form.html.erb]: Do we need `errors_for` in this block?

File: app/views/principals/pre_qualification_form.html.erb [hunk 2]
```diff
@@ -1,47 +1,111 @@
...
+          <div class="form__group-item">
+            <%= f.label :question_1 do %>
+              <%= f.radio_button :question_1, 0, class: 'form__group-input' %>
+              <%= t('registration.answer_no') %>
+            <% end %>
+          </div>
+        </fieldset>
+      <% end %>
+
+      <%= f.form_row :question_2 do %>
```
Comment (benbarnett) [inline app/views/principals/pre_qualification_form.html.erb]: Ditto here and for all questions

File: app/views/principals/pre_qualification_form.html.erb (hunk 1 above)
Comment (benbarnett) [inline app/views/principals/pre_qualification_form.html.erb]: Appreciate they're radio buttons but belt and braces - I'd imagine if none were selected (somehow, or if defaults changed later) then the server would return an error

Comment (benlovell) [inline app/views/principals/pre_qualification_form.html.erb]: The validation messages are not displayed in this page so I don't think this is necessary.

Comment (benbarnett) [inline app/views/principals/pre_qualification_form.html.erb]: Cool

**Comment by benbarnett** [comment]
:+1:

**Comment by benlovell** [comment]
We'll pick this up on Monday morning and get the specs in shape as agreed Friday.

**Comment by benlovell** [comment]
:+1:

**Pull request merged**